sormas-app/app/src/main/java/de/symeda/sormas/app/contact/edit/ContactNewFragment.java

```diff
@@ -104,6 +106,14 @@ public void onLayoutBinding(FragmentContactNewLayoutBinding contentBinding) {
 			initialCommunities,
 			record.getCommunity());
 
+		contentBinding.contactMultiDayContact.addValueChangedListener(e -> {
```

Comment:
showing/hiding firstContact date can be solved in a more simple way using `app:dependencyParentField` and `app:dependencyParentValue` properties in the layout xml